packages/snaps-utils/src/index.browser.ts

```diff
@@ -24,3 +24,4 @@ export * from './types';
 export * from './validation';
 export * from './versions';
 export * from './virtual-file/index.browser';
+export { assertLinksAreSafe } from '@metamask/snaps-ui';
```

Comment:
Why does this need to be exported from `@metamask/snaps-utils`? Can't we just import it from `@metamask/snaps-ui` directly?

Reply:
We can, it's just that `assertLinksAreSafe` are not directly related to Custom UI but defined there to avoid code duplication (since we can't import `snaps-utils` in `snaps-ui`) and was more in-context when imported from `snaps-utils`

Reply:
But we can just get it from `snaps-ui`

Reply:
We currently sometimes import from `snaps-ui` and sometimes from `snaps-utils`. We should standardize.

Reply:
We have `assertLinksAreSafe` and `assertUiLinksAreSafe`, the second one uses the first one but is meant to be used with custom UI. The first one is used for Notifications which only allow a string of text

Reply:
I'm fine with just importing from `snaps-ui` as a way to standardize. I think its more confusing to import from two different places